meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-tools_2.6.bb

```diff
@@ -0,0 +1,31 @@
+SUMMARY = "Cert_create & Fiptool for fip generation for Trusted Firmware-A"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://license.rst;md5=1dd070c98a281d18d9eefd938729b031"
+
+SRC_URI = "git://github.com/ARM-software/arm-trusted-firmware.git;protocol=https;branch=master \
+           file://0001-tools-allow-to-use-a-root-key-password-from-command-.patch \
+           file://0002-fix-fiptool-respect-OPENSSL_DIR.patch \
+           "
+
+SRCREV = "a1f02f4f3daae7e21ee58b4c93ec3e46b8f28d15"
```

Comment:
Have you tried to switch to 2.7 release?

Reply:
not yet but I am going to try

Reply:
add the 2.7 version

Reply:
meta-st upstream prefere to stay with the 2.6 so we can do the bump in our side